core/script_debugger_remote.cpp

```diff
@@ -532,56 +553,77 @@ void ScriptDebuggerRemote::_send_object_id(ObjectID p_id) {
 	if (!obj)
 		return;
 
-	List<PropertyInfo> pinfo;
```

Comment:
I actually like my code for handling requesting objects better. It's newer than yours, fixes a few bugs,  sends Object IDs and can handle when an object is too big to send.

Reply:
I remember you have changed the limit to 8MB it should be enough for most objects. I will check your code again and make it more reliable with this. But it is necessary to change something to send more usable informations like script constant values and member variables of script instance.

Reply:
member variables of script instance should already be obtainable with get_property_list to the object, they have the usage flag: PROPERTY_USAGE_SCRIPT_VARIABLE , so no extra code is needed.